ci-operator/config/open-cluster-management/observatorium-operator/open-cluster-management-observatorium-operator-main.yaml

```diff
@@ -21,48 +21,30 @@ tag_specification:
   namespace: ocp
 test_binary_build_commands: "true"
 tests:
-- as: e2e-test
+- as: e2e-kind
   steps:
-    cluster_profile: aws
+    env:
+      AWS_CREDENTIALS_SECRET: ocm-obs-china-aws-kind
+      AWS_INSTANCE_TYPE: t3.small
+    post:
+    - ref: ocm-e2e-kind-destroy
+    pre:
+    - ref: ocm-ci-rbac
+    - ref: ocm-e2e-kind-create
     test:
-    - as: deploy-operator
-      cli: latest
-      commands: |
-        export OPERATOR_IMAGE_NAME=${IMAGE_FORMAT//\/stable:\$\{component\}//pipeline:observatorium-operator}
-        ./tests/e2e.sh deploy-operator
-      env:
-      - default: oc
-        name: KUBECTL
+    - as: test
+      commands: "echo \"OPERATOR_IMAGE_NAME=$OPERATOR_IMAGE_NAME\"\n./tests/e2e.sh
+        kind\n./tests/e2e.sh deploy-operator\n./tests/e2e.sh test --tls \n./tests/e2e.sh
+        delete-cr\n"
```

Comment:
nit: translate the `command` to multiple-line string to make it easy to read.

Reply:
It seems it was changed if you run `make ci-operator-config`.